spec/Appendix B -- Grammar Summary.md

```diff
@@ -140,10 +140,12 @@ Selection :
   - FragmentSpread
   - InlineFragment
 
-Field : Alias? Name Arguments? Directives? SelectionSet?
+Field : Alias? Name Arguments? Nullability? Directives? SelectionSet?
 
 Alias : Name :
 
+Nullability : (`?`|`!`)?
```

Comment:
Am I doing this notation correct? The vertical bar to indicate the selection between multiple options isn't described in Appendix A, but it is described in the wiki page for [Backus Naur form context free grammar](https://en.wikipedia.org/wiki/Backus%E2%80%93Naur_form#Introduction). I can add that to Appendix A if that would be helpful.